llvm/unittests/Transforms/Utils/MergeFunctionsTest.cpp

```diff
@@ -0,0 +1,271 @@
+//===- MergeFunctionsTest.cpp - Unit tests for
+// MergeFunctionsPass-----------===//
```

Comment:
Resolved in [eae748a](https://github.com/llvm/llvm-project/pull/111045/commits/eae748a33a7c96230cb392d4fe8352af760a687f).

Reply:
clang format keeps changing that to two lines: [1fdf94e](https://github.com/llvm/llvm-project/pull/111045/commits/1fdf94e3e77c25156e56db0d8cde5f7965fb7e2f).

Reply:
Clang-format will format to a max column with of 80 - for the file header comments, you should change the padding hyphens so that it's exactly 80 characters wide (example in the github suggestion).

Reply:
Oosp, my bad. Fixed it! [d193a63](https://github.com/llvm/llvm-project/pull/111045/commits/d193a639f05ad094987f61e2e59dd11daf234891)